.github/workflows/docker-build-publish-prod.yml

```
@@ -6,12 +6,13 @@ on:
 
 jobs:
     ever-api-starter-kit-api:
-        runs-on: ubuntu-22.04
+        runs-on: ${{ vars.RUNNER_LINUX_X64_4 || 'ubuntu-latest' }}
         steps:
             - name: Checkout
               uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)